repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -225,9 +313,9 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
       configuredMethods.contains(HttpMethod.fromValue(requestMethod))
 
   private def getPathsForContentType(contentType: String, resource: HttpMessage): List[LinkPath] = {
-    Option(contentType) match {
-      case Some(ct) if ct.toLowerCase.contains("json") => resource.getJson.toList
-      case Some(ct) if ct.toLowerCase.contains("xml") => resource.getXml.toList map(_.getXpath)
+    (Option(contentType), Option(resource)) match {
```

Comment:
Yes, it was found in testing. And true, it shouldn't happen in production. I just hate to leave a known potential NPE just waiting for something to let it rear it's ugly head.